rebuild-sites-amplify/change-redirs.js

```diff
@@ -0,0 +1,74 @@
+const {
+  AmplifyClient,
+  ListAppsCommand,
+  UpdateAppCommand,
+} = require("@aws-sdk/client-amplify");
+
+const MAX_RESULTS_PER_PAGE = 100
+
+const awsClient = new AmplifyClient({
+  region: "ap-southeast-1",
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+
+async function changeRedirs() {
+  let sitesList = {}
+  let nextToken
+  const listCommand = new ListAppsCommand({
+    maxResults: MAX_RESULTS_PER_PAGE,
+  })
+  response = await (awsClient.send(listCommand))
```

Comment:
are we not missing a `let` or a `const` here

Reply:
Huh you're right, not sure how it was working - fixed in https://github.com/isomerpages/isomer-tooling/pull/44/commits/446ebd92845fb56a9f69b0cdb31fa412f68dee5d